src/util/RouterUtil.js

```diff
@@ -112,7 +112,7 @@ function (Okta, Util, OAuth2Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
         return;
       }
 
-      router.settings.callGlobalSuccess(Enums.SUCCESS, {
+      var successData = {
         user: res._embedded.user,
```

Comment:
Don't know enough about this, just wondering, should we have a type in successData also for this case to be more consistent?